src/sentry/conf/server.py

```diff
@@ -792,6 +793,13 @@ def create_partitioned_queues(name):
         ),
         "options": {"expires": 60 * 60 * 3},
     },
+    "schedule-weekly-organization-reports-new": {
+        "task": "sentry.tasks.weekly_reports.schedule_organizations",
+        "schedule": crontab(
+            minute=0, hour=12, day_of_week="monday"  # 05:00 PDT, 09:00 EDT, 12:00 UTC
+        ),
+        "options": {"expires": 60 * 60 * 3},
```

Comment:
Could we start with a low expiration and set up an alert to see things failing first?

Reply:
@armenzg What happens if the job takes 3+ hours? We'd send duplicate emails?